### DISCUSSION TITLE
Session status not changing

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#130] [Q&A]
Why is my session status not changing from STARTING to SCAN_QR_CODE? It is stuck on STARTING for days and I'm not getting a perfect QR Code when I call the get screenshot endpoint.
Also, does my network speed affect how long it would take to generate a QR Code?

Thanks

**Comment by allburov** [comment]
Hi! Do you use default engine and the latest version of docker image? Is it core version or plus one?
If you start a different session - does it work?
Is there anything in docker logs that could help us?

**Reply by JoshuaBarawa** [reply]
Hi! I'm using the default engine(WEBJS) and the latest docker version. I'm using the Plus version of Waha.
Starting different sessions work but the previous one is still on "STARTING".

**Comment by allburov** [comment]
Could you try to logout from the session with post /api/sessions/logout request and then stop it with /stop and then start again?
Looks like something wrong in the session data

What about logs, could you send me logs for dockef container for that session when you/start it?

**Reply by JoshuaBarawa** [reply]
The docker image is hosted in our prod server. I will send the logs when I get access.
What about generating the QR Code, does my network speed affect how long it would take?

**Reply by allburov** [reply]
It should not. Only if you have like 100kb/s channel or using 3G

**Reply by allburov** [reply]
Actually, it works with my 3G mobile network

**Reply by JoshuaBarawa** [reply]
Okay, thanks. Stopping and logging out of the session helped solve the problem
